docs/tinlake/overview/pricing_rwa.md

```diff
@@ -1 +1,78 @@
+
+A typical freight forwarder invoice is about USD $5,000 with payment due in 30 days. ConsolFreight has embedded itself into the payment systems of hundreds of retailers and freight forwarders. A freight forwarder can electronically request financing from ConsolFreight for an invoice that is already identified in the system. The invoice will serve as collateral and ConsolFreight will collect from the shipper.
+
+To set interest and advance rates, ConsolFreight has two approaches. Often they hire a trade finance insurance company that evaluates and pre-approves every shipper before providing insurance coverage. In other cases, they may also underwrite based on their own risk scorecard. For this example, we will use a highly simplified version of one of ConsolFreight’s scorecards to match the pricing factors we have described above. We further assume that each of the five individual factors described above contributes equally with a score from 1 (the worst) to 10 (the best) to an overall risk score of up to 50.
+
+Now let’s assume we have a shipment from a grocery supplier (“Merchandise Co”) to a supermarket (“Supermarket 24”) in Panama handled by freight forwarder “Tech Cargo”. Tech Cargo invoices Supermarket 24 and requests financing from ConsolFreight. Here is what the scores for an invoice from Tech Cargo may look like:
+
+![](pricing-rwa-2.png)
+
+Adding up the individual scores for each of these factors leads to a risk score of 42 for Tech Cargo.
+
+This risk score can be mapped on a risk scorecard with assigned advance and interest rates for risk scores. The typical advance rate is usually between 85% to 95%. As of the writing of this blog, annual rates (APR) around 10%-15% on a loan backed by a 30-day invoice are consistent with observable market rates. Having this in mind, a simple scorecard could look like this:
+
+![](pricing-rwa-3.png)
+
+Tech Cargo would be assigned a risk rating of B and ConsolFreight would offer Tech Cargo financing with an advance rate of 95% and an interest rate/fee of 12.00% against the invoice as collateral. If the invoice is $5,000, then ConsolFreight would advance $4,704 (95% advance less 12% interest) and collect $4,750 in 30 days from Tech Cargo.
+
+## Outlook
+Let’s take a step further and assume this invoice would be financed through Tinlake. ConsolFreight would be connected to a Tinlake pool. The financed invoice would be tokenized into an NFT, and this NFT could be used as collateral to draw a loan from a DeFi lending protocol.
+
+The crucial question for DeFi is, ***How can these pricing parameters be brought on-chain for real-world assets to be priced and used as collateral?*** This question and others related to pricing will be addressed in subsequent blog posts describing how Tinlake pricing oracles will work and enable decentralized, public pricing of real-world assets as collateral in DeFi.
```

Comment:
This writing style doesn't fit a more static documentation I think, I think we can get rid of this section.